articles/Choosing-between-Django-Flask-and-FastAPI/index.md

```diff
@@ -0,0 +1,79 @@
+Most developers who start off learning Python are often confused after the introductory Python course if they should learn Django, Flask, or Fast API. This article will highlight some features that will enable a beginner to make the right choice.
+
+### Choosing between Django Flask and FastAPI
+
+
+
+![Simple](https://github.com/jamessandy/engineering-education/blob/new-article/articles/prototyping-machine-learning-models-with-streamlit/hero.jpg)
+
+Python is a very beginner-friendly language with a great community. After learning Python, you would need to know a Python framework to work with Python efficiently. A framework is a collection of packages and modules used to develop software and help developers avoid handling low-level details. The three major Python frameworks used in developing web applications are Django, Flask, and Fast API. They come in handy in various use cases. We are going to evaluate each of them to help you make a decision.
+
+#### Tabel of Content
+1. Django Pros & Cons
+2. Flask Pros & Cons
+3. FastAPI Pros & Cons
+4. Comparison
+5. Conclusion
+
+
+#### Django
+Django is a free and open-source Python web development framework used in building websites. It was created by Adrian Holovaty and Simon Willison in 2003 and made use of the Model-Template-View pattern. It is trendy for its robust and straightforward nature. It is one of the most popular frameworks globally and is used by Tech giants like Instagram, Youtube, etc. Django is popular because it encourages reusable code.
+
+##### Pros
+1. Standardized Structure: Django makes its code structure very efficient, which makes it easy for developers to add more functionalities to their websites
+2. REST Framework: The Rest framework for Django, which is DRF, is used for building APIs. It’s easy to build Web APIs because of it’s modular and customizable architecture.
+3. Security: Django emphasizes security by providing defense against joint SQL injection and Cross-site Request forgery attacks.
+
+##### Cons
+1. Django’s slow development: It’s cumbersome software limits the speed of development because of the several reusable modules by the community, and also it needs to make sure that previous versions are still compatible with new releases, which contribute to its slow nature. 
+2. Django ORM: Django models can save themselves and transactions of the default because it was created before SQLAlchemy came into existence.
+
+##### Django: Use cases
+You can use Django to tackle projects of any size and capacity, from simple sites to high-performance sites. It can be used for high load booking engines or shopping platforms, Documentary management systems, Built-in custom CRM systems for internal data, and even IOS and Andriod applications that support web-based applications. Some developer features that can be efficiently implemented with Django is Admin dashboards, Photo-based verification, Emailing systems for sending user notifications, etc.
```

Comment:
I don't think you need to capitalize words in between the sentence. It can be written as Django is admin dashboards, photo-based verification... Change this throughout the article. I won't be picking each instance of capitalization for optimum usage of comments space.